src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -210,38 +262,20 @@ const GuideStore = Reflux.createStore({
     }
 
     // Pick the first guide that satisfies conditions.
-    let bestGuideKey = null;
-    let user = ConfigStore.get('user');
-    for (let key of guideKeys) {
-      if (key === ALERT_REMINDER_1) {
-        if (!this.checkAlertTipData()) {
-          // Wait for the required data.
-          break;
-        } else {
-          let projectId = parseInt(this.state.project.id, 10);
-          if (
-            this.state.projectStats[projectId] > 1000 &&
-            !this.state.projectRules[projectId]
-          ) {
-            bestGuideKey = key;
-            break;
-          }
-        }
-      } else if (user.isSuperuser || new Date(user.dateJoined) > new Date(2018, 4, 10)) {
-        bestGuideKey = key;
-        break;
-      }
-    }
+    let bestGuideKey = await this.getBestGuideKey(guideKeys);
 
     let bestGuide = null;
     if (bestGuideKey) {
-      bestGuide = $.extend(true, {}, this.state.guides[bestGuideKey]);
```

Comment:
We shouldn't use `jQuery` to extend, object spread is preferred (and `lodash` if you need to deep clone).